core/shared/src/main/scala/zio/internal/metrics/package.scala

```diff
@@ -0,0 +1,10 @@
+package zio.internal
+
+package object metrics {
+
+  private[zio] lazy val metricState: ConcurrentState =
+    new ConcurrentState
+
+  val dblOrdering: Ordering[Double] =
```

Comment:
???

Reply:
Scala 2.13 and later have incompatible implementations of this so we need to provide our own.

Reply:
Right now it is `???` implementation.

Reply:
Yes, was just trying to get it to compile. Will update shortly.

Reply:
Implemented now.